### PR TITLE
Fixes build with mingw-w64

### PR DESCRIPTION
**Pull request by tamasmeszaros**
#### Check all that apply (change to `[x]`)
- [x ] All changes meet [libigl style-guidelines](https://libigl.github.io/style-guidelines/).
- [ ] Adds new .cpp file.
- [ ] Adds corresponding unit test.
- [ ] Adds corresponding python binding.
- [ x] This is a minor change.

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+
+...
 
 #### Check all that apply (change to `[x]`)
 - [ ] Windows
 - [ ] Mac OS X
 - [ ] Linux
 
-See <https://libigl.github.io/CONTRIBUTING/#bugreport> for more tips.
+See https://libigl.github.io/CONTRIBUTING/#bugreport for more tips.
```
Comment (teseoch) [inline .github/issue_template.md:10]: why these changes?

Comment (jdumas) [reply .github/issue_template.md:10]: The branch probably duplicates an old commit. The new version after merged is unchanged.

Comment (jdumas) [reply .github/issue_template.md:10]: @teseoch can we merge this?
